docs/platform/webcontainers/browser-support.md

```diff
@@ -56,20 +70,19 @@ Learn [how to configure Brave to run WebContainers](/platform/webcontainers/brow
 
 We have beta support for Firefox. Please try it and [share feedback with us](https://github.com/stackblitz/webcontainer-core/issues/new/choose)!
 
-As mentioned above, Firefox does not fully support the required mode for cross-origin isolation, so you might encounter limitations when **running a server in a preview window** within the StackBlitz editor. Third-party assets might get blocked due to the limitations of cross-origin isolation policies. However, you can work around this by opening your server preview in a separate window.
+Firefox does not fully support the required mode for cross-origin isolation, so you might encounter limitations when **running a server in a preview frame** within the StackBlitz editor. Third-party assets might get blocked due to the limitations of cross-origin isolation policies. However, you can work around this by opening your server preview in a separate window.
 
-In addition to this, there might be [other runtime incompatibilities](#runtime-differences) as detailed above.
+Because Firefox and Node.js use different JavaScript engines (SpiderMonkey and V8), there is a small risk of running into [JS engine differences](#js-engine-differences).
```

Comment:
```suggestion
Because Firefox and Node.js use different JavaScript engines (SpiderMonkey and V8, respectively), there is a small risk of running into [JS engine differences](#js-engine-differences).
```